CMakeLists.txt

```diff
@@ -626,6 +629,10 @@ if ((${CMAKE_SYSTEM_PROCESSOR} MATCHES "arm") OR (${CMAKE_SYSTEM_PROCESSOR} MATC
             # Raspberry Pi 3, 4, Zero 2 (32-bit)
             add_compile_options(-mno-unaligned-access)
         endif()
+	if (LLAMA_ARM_MMLA)
+            add_compile_options(-march=armv8.2-a+i8mm)
```

Comment:
Ah, I was surprised not to see it enabled on Neoverse-N1, but forcing it yielded an invalid instruction, so it look like N1 doens't have i8mm. I was hoping, but no luck here :-/

Reply:
yes, i8mm isn't supported on Neoverse-N1, but you could try Neoverse-V1 (AWS Graviton3,[ c7g instances](https://aws.amazon.com/ec2/instance-types/c7g/)) that has the support.

Reply:
Yeah sure, but I was checking since I have N1 at work :-)  And yes, Graviton3 is awesome, the first time I tested it for other stuff, I thought it would make a great platform to run llama.cpp ;-)